server/client-shared/src/test/scala/cool/graph/private_api/finder/CachedProjectFetcherImplSpec.scala

```diff
@@ -56,7 +56,7 @@ class CachedProjectFetcherImplSpec extends FlatSpec with Matchers with ScalaFutu
     projectFetcher.setAlias(firstAlias = None, secondAlias = None)
     pubSub.publish(Only("FirstOne"), "FirstOne")
 
-    Thread.sleep(2000)
+    Thread.sleep(3000)
```

Comment:
Didn't you do this in a previous PR as well?

Reply:
See https://github.com/graphcool/framework/pull/1096/files/d410409b82bc14133aea2e5b91e362f90e003b3c#diff-7f784c4dd4c366679489718601b496f3R61